Design Patterns/Abstract Factory Pattern/src/GPUs/AmdMsiGPU.java

```diff
@@ -0,0 +1,10 @@
+package GPUs;
```

Comment:
please follow naming convention for packaging: like: com.exalt.training.designpattern

please apply it for all